services/harmony/test/ignore-errors.ts

```diff
@@ -22,14 +22,14 @@ const reprojectAndZarrQuery = {
   scaleExtent: '0,2500000.3,1500000,3300000',
   scaleSize: '1.1,2',
   format: 'application/x-zarr',
-  concatenate: false,
+  concatenate: 'false',
```

Comment:
Why were these changes needed? Doesn't the http client library send boolean values in as the strings true or false?

Reply:
You are right, no need to change here since this didn't get submitted as POST. The other places where it is submitted as POST body still need to be changed. I will change it back here. Thanks!